tensorflow/workspace.bzl

```diff
@@ -169,11 +169,10 @@ def tf_workspace(path_prefix = "", tf_repo_name = ""):
     tf_http_archive(
         name = "libxsmm_archive",
         build_file = clean_dep("//third_party:libxsmm.BUILD"),
-        sha256 = "cd8532021352b4a0290d209f7f9bfd7c2411e08286a893af3577a43457287bfa",
-        strip_prefix = "libxsmm-1.9",
+        sha256 = "c6240d0df38e729da5cb6d0f1e11a755507649ee5a94727bb0c47549309c35b6",
+        strip_prefix = "libxsmm-1ea106130554f252669015aa7613b1f05f42b723",
         urls = [
-            "http://mirror.tensorflow.org/github.com/hfp/libxsmm/archive/1.9.tar.gz",
-            "https://github.com/hfp/libxsmm/archive/1.9.tar.gz",
+            "https://github.com/hfp/libxsmm/archive/1ea106130554f252669015aa7613b1f05f42b723.zip",
```

Comment:
Please also add the mirror link. I'll add the file to the mirror later. (Bazel won't fail even if the link doesn't exist now.)
```
"http://mirror.tensorflow.org/github.com/hfp/libxsmm/archive/1ea106130554f252669015aa7613b1f05f42b723.zip
```

Reply:
Update: I've put the file on the mirror. The mirror URL works now.